src/gui/parameters_screen.py

```diff
@@ -0,0 +1,413 @@
+            create_label_grid(cells_side_count_latitude_frame, "Data stations count:\nlatitude")
+            create_label_grid(cells_side_count_longitude_frame, "Data stations count:\nlongitude")
+            create_label_grid(point_side_size_frame, "Point side size\n[m]")
+
+            self.top_coord_input = create_input_entry_grid(top_coord_frame, 9, str(self.top_coord),
+                                                           self.validate_coordinates_top)
+            self.down_coord_input = create_input_entry_grid(down_coord_frame, 9, str(self.down_coord),
+                                                            self.validate_coordinates_down)
+            self.left_coord_input = create_input_entry_grid(left_coord_frame, 9, str(self.left_coord),
+                                                            self.validate_coordinates_left)
+            self.right_coord_input = create_input_entry_grid(right_coord_frame, 9, str(self.right_coord),
+                                                             self.validate_coordinates_right)
+            self.time_range_start_input = create_input_entry_grid(time_range_start_frame, 17,
+                                                                  str(self.time_range_start),
+                                                                  self.validate_time_range_start)
+            self.time_range_end_input = create_input_entry_grid(time_range_end_frame, 17, str(self.time_range_end),
+                                                                self.validate_time_range_end)
+            self.data_time_step_input = create_input_entry_grid(data_time_step_frame, 3,
+                                                                str(self.data_time_step_minutes),
+                                                                self.validate_data_time_step)
+            self.cells_side_count_latitude_input = create_input_entry_grid(cells_side_count_latitude_frame, 3,
+                                                                           str(self.cells_side_count_latitude),
+                                                                           self.validate_cells_side_count_latitude)
+            self.cells_side_count_longitude_input = create_input_entry_grid(cells_side_count_longitude_frame, 3,
+                                                                            str(self.cells_side_count_longitude),
+                                                                            self.validate_cells_side_count_longitude)
+            self.point_side_size_input = create_input_entry_grid(point_side_size_frame, 3, str(self.point_side_size),
+                                                                 self.validate_point_side_size)
+
+            self.top_coord_validation_label = create_label_grid(top_coord_frame, "Default value", 2, 0,
+                                                                ("Arial", 8, "bold"))
```

Comment:
there is a lot of code that could use some default parameters. Maybe function create Parameter screen label? that only takes the frame?